storybook/webpack.config.ts

```diff
@@ -112,11 +112,12 @@ export const elm: Rule = {
       loader: "elm-hot-webpack-loader",
     },
     {
-      loader: "elm-webpack-loader",
+      loader: resolve(__dirname, "elm-webpack-loader-fix.js"),
       options: {
         debug: false,
         cwd: resolve(__dirname, ".."),
         pathToElm: resolve(__dirname, "../node_modules/.bin/elm"),
+        forceWatch: true,
```

Comment:
This was the first bug I found. Normally you don't need to include this, because node-elm-compiler infers that webpack is in watch mode by parsing the command line arguments (https://github.com/elm-community/elm-webpack-loader/blob/master/index.js#L79), but Storybook does not use the dev server, it has it's own custom server, presumably with watch mode configured somewhere. Fixing this caused the problem where too many files were being watched.

Reply:
We needed to `forceWatch` in Murmur for awhile, when we were using a custom programmatic invocation of webpack-dev-server rather than the standard NPM binary for this. Being able to remove `forceWatch` from our Elm loader was one of the main drivers for switching to "standard" webpack-dev-server. I understand this isn't an option for Kaizen's webpack config, since it's all wrapped in Storybook.